spec/rundeck/client/system_info_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'spec_helper'
+
+describe Rundeck::Client do
+  describe '.system_info', vcr: { cassette_name: 'system_info' } do
+    before do
+      @system_info = Rundeck.system_info
+    end
+
+    it 'expects a get to have been made' do
+      expect(a_get('/system/info')).to have_been_made
+    end
+
+    it '@system_info reponse is expected to be a Objectified Hash' do
+      expect(@system_info).to be_a Rundeck::ObjectifiedHash
+    end
+
+    describe '#system_info system block response validation' do
+
+      it 'rundeck block response' do
+        expect(@system_info.system.rundeck.version).to eq("2.4.2")
+        expect(@system_info.system.rundeck.build).to eq("2.4.2-1")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.